packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -77,6 +79,11 @@ export class TriggerLauncher extends Component {
     this.setState({ status: TWO_FA_MISMATCH, showTwoFAModal: true })
   }
 
+  handle2FACode(code) {
```

Comment:
Should definitely not go here. That's why I duplicated the `TwoFAForm` on the first place.

Reply:
But the duplicate didn't work as well. What's the difference of passing a props and passing a method here?

Reply:
Not relevant anymore